public/components/datasources/components/manage/accelerations/utils/acceleration_utils.tsx

```diff
@@ -103,7 +104,7 @@ export const CreateAccelerationFlyoutButton = ({
     <>
       <EuiButton
         onClick={() =>
-          renderCreateAccelerationFlyout(dataSourceName, undefined, undefined, handleRefresh)
+          renderCreateAccelerationFlyout(dataSourceName, '', undefined, undefined, handleRefresh)
```

Comment:
`dataSourceMDSId?: string` -> Shouldn't the parameter here be undefined, rather than empty string?  Similar to how we are doing it for databaseName and tableName.